_config.yml

```diff
@@ -10,11 +10,11 @@ title: GNU Octave
 email: maintainers@octave.org
 description: > # this means to ignore newlines until "baseurl:"
   GNU Octave is a programming language for scientific computing.
-baseurl: "/software/octave/new" # the subpath of your site, e.g. /blog
-url: "https://www.gnu.org" # the base hostname & protocol for your site
-wiki_url: "http://wiki.octave.org"
-faq_url: "http://wiki.octave.org/faq"
-docs_url: "https://www.gnu.org/software/octave/doc/interpreter/"
+baseurl: "/octave-web" # the subpath of your site, e.g. /blog
+url: "//www.gnu.org" # the base hostname & protocol for your site
+wiki_url: "//wiki.octave.org"
```

Comment:
The version of my pull request can be seen here: https://www.gnu.org/software/octave/new/

The `//` syntax I used, because it was provided in your template. Is it preferable to use protocol-less URLs? https://stackoverflow.com/questions/4831741/can-i-change-all-my-http-links-to-just